lib/spdy/server.js

```diff
@@ -189,6 +189,8 @@ proto._onStream = function _onStream (stream) {
     socket = new net.Socket(socketOptions)
   }
 
+  if (process.versions.modules >= 46) { socket.server = this }
```

Comment:
Currently this is only needed for Node.js master but also handles the unlikely case where https://github.com/nodejs/node/pull/11926 will be backported to Node.js 6 and 4 (`modules >= 46`).